src/plugins/timeline/TimelineViewLayout.vue

```diff
@@ -101,7 +89,7 @@ function getViewKey(domainObject, objectPath, openmct) {
 
 export default {
     components: {
-        ObjectView,
+        TimelineObjectView,
         TimelineAxis,
         SwimLane
```

Comment:
same, remove this?

Reply:
TimelineObjectView is also being used in this view.